doc/foreign-code.rst

```diff
@@ -193,10 +193,10 @@ and build and link the example ``foo.exe`` program.
 Complete information about the ``ctypes`` combinators used above is available
 at the ctypes_ project.
 
-Ctypes Stanza Reference
+Ctypes Field Reference
```

Comment:
Unfortunately this might break existing links to this section, like from this article: https://michael.bacarella.com/2022/02/19/dune-ctypes/. cc @mbacarella